packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -56,14 +68,32 @@ const bottomMenuItems: SidebarMenuItem[] = [
 export default function Sidebar({
   useNavButtonsNotLinks = false,
   activePage,
+  showSidebar = false,
+  openMobileSidebar = false,
   onNavTabClick,
   enableSearch,
+  setOpenMobileSidebar,
+  onShowDndClick,
 }: SidebarProps): ReactElement {
   const { alerts } = useContext(AlertContext);
+  const { user, showLogin, loadingUser } = useContext(AuthContext);
   const { isLoaded, isAnimated, setLoaded, setHidden } =
     useDynamicLoadedAnimation();
   const { openSidebar, toggleOpenSidebar } = useContext(SettingsContext);
   const [showSettings, setShowSettings] = useState(false);
+  const router = useRouter();
+
+  useEffect(() => {
```

Comment:
We introduced this route change to close the sidebar, because we have actions and links triggering change.

Reply:
This calls for a custom hook 😉